app/models/user.rb

```diff
@@ -14,9 +14,9 @@ class User < ApplicationRecord
   include Blacklight::User
 
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :confirmable, :lockable, :registerable, :recoverable, :timeoutable and :omniauthable
   devise :invitable, :remote_user_authenticatable, :database_authenticatable,
-         :registerable, :recoverable, :rememberable, :trackable, :validatable
+         :rememberable, :trackable, :validatable
```

Comment:
This didn't ultimately solve the problem, but seemed like a good idea?  `/shrug`